dask/dataframe/io/parquet/arrow.py

```diff
@@ -297,10 +305,12 @@ def read_metadata(
                 for i, piece in enumerate(pieces):
                     num_row_groups = row_groups_per_piece[i]
                     for rg in range(num_row_groups):
-                        parts.append((piece.path, rg, piece.partition_keys))
+                        parts.append(
+                            (piece.path, rg, piece.partition_keys, row_group_sizes[rg])
+                        )
             else:
                 parts = [
-                    (piece.path, piece.row_group, piece.partition_keys)
+                    (piece.path, piece.row_group, piece.partition_keys, None)
```

Comment:
I wonder if we might include the size in the statistics along with column min/max and such, rather than extend this data structure.